features/items.md

```diff
@@ -195,27 +211,25 @@ If you want to use the dynamically items just use the image name without the add
 
-```java
-Switch  Light_Floor        "Light at Floor"                { knx="1/0/15+0/0/15" }
-Switch  Presence           "I'm at home"                   { bluetooth="123456ABCD" }
-Switch  Doorbell           "Doorbell"                      { serial="/dev/usb/ttyUSB0" }
-Contact Garage             "Garage is [MAP(en.map):%s]"    { zwave="21:command=sensor_binary,respond_to_basic=true" }
-String  Error_Ventilation  "Error in Ventilation %s"       { comfoair="error_message" }
-Number  DiningRoomTemp     "Maximum Away Temp. [%.1f °F]"  { nest="<[thermostats(Dining Room).away_temperature_high_f]" }
+```xtend
+Switch  Phone_Mobile       "My Mobile Phone"               { channel="network:device:devicename:online" }
+Number  Netatmo_Indoor_CO2 "CO2"                           { channel="netatmo:NAMain:home:inside:Co2" }
+Number  Azimuth            "Azimuth"                       { channel="astro:sun:home:position#azimuth" }
+Contact Garage             "Garage is [MAP(en.map):%s]"    { channel="zwave:21:command=sensor_binary,respond_to_basic=true" }
 ```
 
-When you install a binding through PaperUI it will automatically create a .cfg file in ```${openhab_home}/conf/services/``` for the appropriate binding. Inside these files are a predefined set of variables which are required for the binding to operate. In many cases you will need to view and edit these to suit your system. These variables can hold ip addresses, API keys, user names, passwords etc. These are all in plain text, so be careful who you share these with if some data is sensitive.
+When you install a binding through PaperUI it will automatically create a .cfg file in `conf/services/` for the appropriate binding. Inside these files are a predefined set of variables which are required for the binding to operate. In many cases you will need to view and edit these to suit your system. These variables can hold ip addresses, API keys, user names, passwords etc. These are all in plain text, so be careful who you share these with if some data is sensitive.
 
-If you are using legacy openHAB 1.x ( or beta/custom bindings ), then these .jar files need to be placed in the ```${openhab_home}/addons/``` folder. If configuration is required then you will need to create an `openhag.cfg` file in ```${openhab_home}/conf/services/``` and paste the appropriate binding configuration into this.
+If you are using legacy openHAB 1.x ( or beta/custom bindings ), then these .jar files need to be placed in the `/addons/` folder. If configuration is required then you will need to create an `openhag.cfg` file in `/conf/services/` and paste the appropriate binding configuration into this.
```

Comment:
That's not completely correct. While special bindings from outside sources might need to be places in this special folder, all legacy bindings are available through openhab. To allow the use, check with the legacy parameter in addons.cfg. Also configuration will take pace in bindingname.cfg.
Also you are right, that these parts should go to an extra bindings page. let's leave it in here for now.